### PR TITLE
add: link to Medium

### PR DESCRIPTION
**Pull request by CurlyBracketEffect**
Quick and dirty!
Added a button to the top right of the page called "Updates" that leads people to the Medium.

I hope you can forgive me my prettier ran amuck with the formatting. 

I could add the icon and link to the footer too, but I would need an SVG that I don't have right now.

### CODE REVIEW

File: css/agave.css [hunk 1]
```diff
@@ -177,11 +184,31 @@ p {
   display: block;
   max-width: 1210px;
   margin-right: auto;
-  margin-left: auto;
   background-color: #d3fbed;
 }
 
+.updates {
```
Comment (CurlyBracketEffect) [inline css/agave.css:190]: I added styling here

File: css/agave.css [hunk 2]
```diff
@@ -177,11 +184,31 @@ p {
...
+  text-decoration: none;
+}
+
+.updates:hover {
+  background-color: #017466;
+  box-shadow: 0 0 8px 1px hsla(0, 0%, 98.7%, 0.27);
+}
+
 .navbar {
+  display: flex;
```
Comment (CurlyBracketEffect) [inline css/agave.css:211]: and here

File: css/agave.css [hunk 3]
```diff
@@ -608,7 +635,7 @@ p {
   }
 
   .navbar {
-    display: block;
+    display: flex;
```
Comment (CurlyBracketEffect) [inline css/agave.css:638]: added flex to the navbar

File: index.html [hunk 4]
```diff
@@ -1,168 +1,286 @@
...
+      data-duration="400"
+      role="banner"
+      class="navbar w-nav"
+    >
+      <div class="nav-container w-container">
+        <a href="index.html" aria-current="page" class="w-nav-brand w--current"
+          ><img src="images/logo.svg" loading="lazy" alt="logo" class="image-2"
+        /></a>
+      </div>
+      <a href="https://agavefinance.medium.com/" class="updates">Updates</a>
```
Comment (CurlyBracketEffect) [inline index.html:92]: Here's the new button

**Review by zedkai — APPROVED** [review]
The format pass was a bit rough, but it looks similar enough that I'll let you deal with any cleanup it requires later.

**Pull request merged**